packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -124,4 +128,8 @@
       margin-top: var(--terra-notification-dialog-actions-button-margin-top, 0.4rem);
     }
   }
+
+  .text-wrapper {
+    flex-shrink: 1;
```

Comment:
Might be good to add a inline comment on why this still is needed so we remember.

Reply:
Added here https://github.com/cerner/terra-framework/pull/645/commits/a4dd349cdf9f28036fbc54b1a24a1d2419c09364.